### ISSUE TITLE
Can you release the Loss function related code?

### ISSUE THREAD

**Issue by bang2003er** [nie-lang/StabStitch#6]
Thank you for your wonderful work. Can you release the code of the Loss function?

**Comment by nie-lang** [comment]
We plan to release the training code of the extended version (StabStitch++) in about two weeks. The repository link is https://github.com/nie-lang/StabStitch2. Stay tuned！
